tests/CMakeLists.txt

```diff
@@ -65,6 +65,10 @@ if (UNIX)
     target_link_libraries(ConnectionFailTest ${CLIENT_LIBS} pulsarStatic ${GTEST_LIBRARY_PATH})
 endif ()
 
+add_executable(BrokerMetadataTest brokermetadata/BrokerMetadataTest.cc)
+target_compile_options(BrokerMetadataTest PRIVATE "-DTEST_ROOT_PATH=\"${CMAKE_CURRENT_SOURCE_DIR}\"")
```

Comment:
The `TEST_ROOT` macro is never used, we should remove the additional compile option here, as well as the related code in `BrokerMetadataTest.cc`.

Reply:
@BewareMyPower Fixed